internal/command/user_v2_invite_test.go

```diff
@@ -0,0 +1,1136 @@
+			args{
+				ctx:    context.Background(),
+				userID: "",
+			},
+			zerrors.ThrowInvalidArgument(nil, "COMMAND-Sgf31", "Errors.User.UserIDMissing"),
+		},
+		{
+			"user does not exist",
+			fields{
+				eventstore: expectEventstore(
+					expectFilter(),
+				),
+			},
+			args{
+				ctx:    context.Background(),
+				userID: "unknown",
+			},
+			zerrors.ThrowPreconditionFailed(nil, "COMMAND-HN34a", "Errors.User.NotFound"),
+		},
+		{
+			"code does not exist",
+			fields{
+				eventstore: expectEventstore(
+					expectFilter(
+						eventFromEventPusher(
+							user.NewHumanAddedEvent(context.Background(),
+								&user.NewAggregate("userID", "org1").Aggregate,
+								"username", "firstName",
+								"lastName",
+								"nickName",
+								"displayName",
+								language.Afrikaans,
+								domain.GenderUnspecified,
+								"email",
+								false,
+							),
+						),
+					),
+				),
+			},
+			args{
+				ctx:    context.Background(),
+				userID: "unknown",
+			},
+			zerrors.ThrowPreconditionFailed(nil, "COMMAND-Wr3gq", "Errors.User.Code.NotFound"),
+		},
+		{
+			"sent ok",
+			fields{
+				eventstore: expectEventstore(
+					expectFilter(
+						eventFromEventPusher(
+							user.NewHumanAddedEvent(context.Background(),
+								&user.NewAggregate("userID", "org1").Aggregate,
+								"username", "firstName",
+								"lastName",
+								"nickName",
+								"displayName",
+								language.Afrikaans,
+								domain.GenderUnspecified,
+								"email",
+								false,
+							),
+						),
+						eventFromEventPusher(
+							user.NewHumanInviteCodeAddedEvent(context.Background(),
+								&user.NewAggregate("userID", "org1").Aggregate,
+								&crypto.CryptoValue{
+									CryptoType: crypto.TypeEncryption,
+									Algorithm:  "enc",
+									KeyID:      "id",
+									Crypted:    []byte("code"),
+								},
+								time.Hour,
+								"",
+								false,
+								"",
+								"authRequestID",
+							),
+						),
+					),
+					expectPush(
+						eventFromEventPusher(
+							user.NewHumanInviteCodeSentEvent(context.Background(),
+								&user.NewAggregate("userID", "org1").Aggregate,
+							),
+						),
+					),
+				),
+			},
+			args{
+				ctx:    context.Background(),
+				userID: "userID",
+			},
+			nil,
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			c := &Commands{
+				eventstore: tt.fields.eventstore(t),
+			}
+			err := c.InviteCodeSent(tt.args.ctx, tt.args.userID, tt.args.orgID)
+			assert.ErrorIs(t, err, tt.wantErr)
+		})
+	}
+}
+
+func TestCommands_VerifyInviteCode(t *testing.T) {
+	type fields struct {
+		eventstore     func(*testing.T) *eventstore.Eventstore
+		userEncryption crypto.EncryptionAlgorithm
+	}
+	type args struct {
+		ctx    context.Context
+		userID string
+		code   string
+	}
+	type want struct {
+		details *domain.ObjectDetails
+		err     error
+	}
+	tests := []struct {
+		name   string
+		fields fields
+		args   args
+		want   want
+	}{
+		{
```

Comment:
No test with failed verification?

Reply:
as this function is just a wrapper where no password or user agent i only checked it for coverage 😛 
all test cases are covered in the `TestCommands_VerifyInviteCodeSetPassword` as mentioned here:
https://github.com/zitadel/zitadel/blob/ed8651392f9260db160d967397ad55e96f62d2e2/internal/command/user_v2_invite_test.go#L794